test/e2e_node/density_test.go

```diff
@@ -77,18 +77,16 @@ var _ = SIGDescribe("Density", framework.WithSerial(), framework.WithSlow(), fun
 	})
 
 	f.Context("create a batch of pods", framework.WithFlaky(), func() {
-		// TODO(coufon): the values are generous, set more precise limits with benchmark data
```

Comment:
can you please replace this with the short instruction on how those numbers were calculated and how to update them if needed. Some combination of what you have in PR description and a few extra words would help

Reply:
@SergeyKanzhelev done, PTAL again, thanks!